### PR TITLE
Artist / Investor Resources Updates

### PR DESCRIPTION
**Pull request by RevolutionTech**

### CODE REVIEW

File: templates/extra/resources.html [hunk 1]
```diff
@@ -126,7 +140,7 @@ <h4>Sharing with blogs and media</h4>
             <div id="launch-5">
                 <h4>Campaign doesn't hit goal</h4>
                 <p>
-                    If your campaign doesn’t hit its goal in 30 days, all investors will be refunded their money. We cannot guarantee that every project will be successful, but a new campaign can always be launched after evaluating the previous one. From what we have seen, a few reasons why campaigns end up being unsuccessful include: 1) the valuation being too high (raising too much money and giving too small of a percentage back), 2) your story or campaign is not compelling enough, and 3) the lack of promotion.
+                    If your campaign doesn’t hit its goal in the time that you set, all money invested will be credited back to the investors. We cannot guarantee that every project will be successful, but a new campaign can always be launched after evaluating the previous one. From what we have seen, a few reasons why campaigns end up being unsuccessful include: 1) the valuation being too high (raising too much money and giving too small of a percentage back), 2) your story or campaign is not compelling enough, and 3) the lack of promotion.
```
Comment (RevolutionTech) [inline templates/extra/resources.html]: You could consider using a list here:
```html
<p>
    If your campaign doesn't hit its goal...
    ...end up being unsuccessful include:
</p>
<ol>
    <li>the valuation being too high (raising too much money and giving too small of a percentage back)</li>
    <li>your story or campaign is not compelling enough</li>
    <li>the lack of promotion</li>
</ol>

File: perdiem/urls.py [hunk 2]
```diff
@@ -88,6 +88,11 @@
         TemplateView.as_view(template_name="extra/privacy.html"),
         name="privacy",
     ),
+    url(
+        r"^investor-resources/?$",
```
Comment (RevolutionTech) [inline perdiem/urls.py]: You could make this something like `r"^resources/investor/?$"` and then change the artist resources to be something like `r"^resources/artist/?$"`.

**Comment by RevolutionTech** [comment]
Just noticed that linting is failing due to some Python formatting being off. You can fix this by running `poetry run black .`.

If in the future you are making a lot of changes to the Python part of the codebase, I should probably help you make some changes to your development environment to make this automatic as you work on things.

File: perdiem/urls.py [hunk 3]
```diff
@@ -88,6 +88,11 @@
...
+        TemplateView.as_view(template_name="extra/investor-resources.html"),
+        name="investor-resources",
+    ),
```
[the remark this answers is not in the document]
Comment (localastronaut) [reply perdiem/urls.py]: Yeah, that totally makes sense

File: perdiem/urls.py (hunk 2 above)
Comment (localastronaut) [reply perdiem/urls.py]: Yeah, I think this makes sense. Just realized that some of my base and home updates include these links and are updated in another branch. So I'll just submit another PR later with these changes

**Comment by localastronaut** [comment]
Geeze. I guess my spellcheck wasn't working when I was writing this. Thanks for those catches. I'll try pasting it into another word processor next time to catch those.

**Comment by RevolutionTech** [comment]
This PR looks good now, but it looks like `black` is still failing, as described above.

**Pull request merged**